config/routes.rb

```diff
@@ -2,6 +2,14 @@
 
 Rails.application.routes.draw do
   devise_for :user, controllers: { registrations: "users/registrations" }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
   root to: "root#index"
+
+  get "/articles/new", to: "article#new", as: "new_article"
+  post "/articles/new", to: "article#new", as: "articles"
+  get "/articles/:id/edit", to: "article#edit", as: "edit_article"
+  patch "/articles/:id/edit", to: "article#edit"
+  delete "/articles/:id", to: "article#remove", as: "remove_article"
+  get "/articles", to: "article#index", as: "self_articles_list"
+  get "/articles/:id", to: "article#index", as: "articles_list"
```

Comment:
If you want to add filtering you can go with the same index action on the previous line but using query parameters:
```ruby
link_to "My articles", articles_path(user_id: current_user.id)
```

Or you can create a separate endpoint
```suggestion
  get "/articles/my", to: "article#my", as: "my_articles"
```